tests/testthat/test-test-data.R

```diff
@@ -1,10 +1,12 @@
-test_that("get_daily_test_data() works as expected", {
-  get_daily_test_data()
+test_that("get_test_data() works as expected", {
+  get_test_data()
```

Comment:
I think we could use `write = FALSE` here and just perform checks on the returned object.